src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const list = document.querySelector('.gallery__list');
+
+const largeImg = document.querySelector('#largeImg');
+
+list.addEventListener('click', (even) => {
+  const item = even.target.closest('.list-item__link');
+
+  even.preventDefault();
```

Comment:
```suggestion
list.addEventListener('click', (even) => {
  even.preventDefault();
    
  const item = even.target.closest('.list-item__link');
```

better to call `preventDefault()` right at the beginning of the function